src/accelerate/state.py

```diff
@@ -34,6 +35,20 @@
 except ImportError:
     _tpu_available = False
 
+try:
+    import deepspeed
+
+    _deepspeed_available = True
+except:
+    _deepspeed_available = False
```

Comment:
I would prefer not importing directly (see [here](https://github.com/huggingface/accelerate/blob/a96fbaaf19e650da101bfadb886ecc1d1b1cb205/src/accelerate/utils.py#L31) for an example on how to this).